common/util.go

```diff
@@ -147,17 +147,24 @@ func DefaultProfileName(width int, height int, bitrate int) string {
 	return fmt.Sprintf("%dx%d_%d", width, height, bitrate)
 }
 
-func FFmpegProfiletoNetProfile(ffmpegProfiles []ffmpeg.VideoProfile) ([]*net.VideoProfile, error) {
+func FFmpegProfiletoNetProfile(ffmpegProfiles []ffmpeg.VideoProfile) ([]*net.VideoProfile, bool, error) {
```

Comment:
I'm not thrilled with this change - it is a bit "leaky" in that it's quite specifically catering to the caller's need to know which `net.SegData.FullProfile/2` field to use within `genSegCreds`, not to mention the sig change to all the test cases - but the alternative is to do another iteration through the set of profiles in `net.SegData` .

Happy to make changes if someone has a strong preference for another way.

Reply:
Fixed in https://github.com/livepeer/go-livepeer/pull/1439/commits/f27bef80a7de76bf591d737cbda7231c97826eec